README.md

```diff
@@ -51,17 +33,6 @@ curl -d "user[email]=user@example.com" -d "user[password]=password" -d "user[pas
 
 When you successfully create a user you'll get a token in the json response, use that in any further communications with the API, you need to set a request header with `Authorization` and use the generate auth_token.
 
-## Delete user
```

Comment:
Well, I guess someone should be able to delete a user, but with the current setup, the only one who can do that is the one with the token. Maybe we should be the ones creating and deleting users (and thus keeping tabs on the tokens), instead of delegating that to the applicants? That would also give us better control of who's actually working on the assignment at any given time?